core/jobs/batch_jobs/tabs_collapsables_validation_jobs.py

```diff
@@ -0,0 +1,582 @@
+                        'caption attr not exists'
+                    )
+
+            elif tag.name == 'oppia-noninteractive-math':
+                try:
+                    if tag['math_content-with-value'] in (
+                        '&quot;&quot;', '', '\'\'', '\"\"'):
+                        states_with_errored_values.append(
+                            'math content attr is empty'
+                        )
+                    try:
+                        math_content = json.loads(
+                            tag['math_content-with-value'])
+                        if math_content['raw_latex'].strip() in (
+                                '&quot;&quot;', '', '\'\'', '\"\"'):
+                            states_with_errored_values.append(
+                                'raw lattex attr empty'
+                            )
+                    except Exception:
+                        states_with_errored_values.append(
+                            'raw lattex attr not exists'
+                        )
+
+                    try:
+                        math_content = json.loads(
+                            tag['math_content-with-value'])
+                        svg_filename = math_content['svg_filename']
+                        if svg_filename.strip() in (
+                            '&quot;&quot;', '', '\'\'', '\"\"'):
+                            states_with_errored_values.append(
+                                'svg_filename attr empty'
+                            )
+                        elif svg_filename.strip().split('&quot;')[1] != '.svg':
+                            states_with_errored_values.append(
+                                'svg_filename attr does not have svg extension'
+                            )
+
+                    except Exception:
+                        states_with_errored_values.append(
+                            'svg_filename attr not exists'
+                        )
+
+                except Exception:
+                    states_with_errored_values.append(
+                        'math content attr not exists'
+                    )
+
+            elif tag.name == 'oppia-noninteractive-skillreview':
+                try:
+                    if tag['text-with-value'].strip() in (
+                        '&quot;&quot;', '', '\'\'', '\"\"'):
+                        states_with_errored_values.append(
+                            'text attr empty in skillreview tag'
+                        )
+
+                except Exception:
+                    states_with_errored_values.append(
+                        'text attr not in skillreview tag'
+                    )
+
+            elif tag.name == 'oppia-noninteractive-video':
+                try:
+                    tag['start-with-value']
+
+                except Exception:
+                    states_with_errored_values.append(
+                        'start value invalid'
+                    )
```

Comment:
I tried doing that previously, we won't be able to do that because here the `tag` is not considered as a string it is a value that is of type from `bs4`. It will always go the `else` part.
Thanks

Reply:
Ah I see thanks for clarifying. Bs4 might have a class method that does this operation?

Reply:
Yeah

Reply:
Have you found if there's a method for this?

Reply:
Yes, the type of the tag is `bs4.element.Tag`

Reply:
No, I mean Bs4 may have a cleaner method to call. Like tag.contains('start-with-value'). So then instead of try except you can do an if statement, which is better Python.

Reply:
I see, my bad. I have found one method which is `tag.has_attr()`. Will it be okay if I use this thing in the conversion function instead, otherwise there will be lots of changes?
Thanks